developer-docs/bolt-server.md

```diff
@@ -200,10 +200,13 @@ curl -v --cacert $BOLT_CACERT --cert $BOLT_CERT --key $BOLT_KEY $BOLT_ROOT
 
 ## Running in a container
 
-from your checkout of bolt build and run the container with
+from your checkout of bolt start the spec dokcer-compose to run
+puppet-server and some targets then run start the top level compose to start
+bolt-server connected to that network.
 
 ```
-docker run --rm -p 62658:62658 `docker build -q ./`
+docker-compose docker-compose -f spec/docker-compose.yml up -d --build
```

Comment:
I'm confused, is this what you're actually supposed to run?

Reply:
there appear to be some larger problems with it.